settings.gradle

```diff
@@ -14,3 +14,4 @@ dependencyResolutionManagement {
 }
 rootProject.name = "MoviesAndroidApp"
 include ':app'
+include ':MovieDetails'
```

Comment:
I think by convention the module name is written in lowercase.
Could you please change this? :)